portality/models/background.py

```diff
@@ -25,6 +34,9 @@ def __init__(self, **kwargs):
 
         super(BackgroundJob, self).__init__(raw=kwargs)
 
+        if not self.outcome_status:
+            self.outcome_status = OutcomeStatus.Success
```

Comment:
I would leave this unset if it is not provided

Reply:
any suggestion for where to mark as Success ?
if not on __init__ ,  `BackgrounApi.execute` could be another choice. better

Reply:
Ah yes, I see the problem.  Perhaps on `save` if it has not already been set?  I suppose it doesn't make much difference, perhaps easiest to leave it here in `init`